src/App.jsx

```diff
@@ -1,86 +1,112 @@
-import {useState, useEffect} from "react";
+import { useState, useEffect } from "react";
 import "./App.css";
 
+const initialTodo = { text: "Iniciar con buena actitud", completed: false };
+
 function App() {
-    const [todos, setTodos] = useState(()=> {
-        const storedTddos = localStorage.getItem('todos');
-        return storedTddos ? JSON.parse(storedTddos) : [{ text: 'Iniciar con buena actitud', completed: false }];
-    });
-    const [newTodo, setNewTodo] = useState('');
+  return <TodoList />;
+}
+
+function TodoList() {
```

Comment:
I took the liberty to rename the `<App />` as `<TodoList />`

i'd definitively create a component named <TodoList />, so the <App> itself is as easy as: 

```js
function App() {
  return <TodoList />
}
```

so later (exercise no.3) will be easier to render the TodoList **or** the Login form

_guiño guiño ;) ;) ;) ;)_